CONTRIBUTING.md

```diff
@@ -552,113 +544,12 @@ example-alert-condition2.yml
 
 For documentation on the rest of the alert condition fields, please review the [Introduction to alerts](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts)
 
-### Install plans
```

Comment:
Let's not remove this, because all the related code and configurations are still living within the ecosystem. Instead we can add `Install plans (DEPRECATED)` and toss a small blurb underneath that explains the changes and points them over to data sources.

Reply:
I see that you've moved a lot of this content down, so I think we're covered. The only reason I could see leaving this section is to keep the docs fairly consistent, that way users don't get confused when they come back to this after these changes go in. Just something to think about, I think we're covered either way.

Reply:
Hello!! yes it is still there, it's just moved underneath data sources because I thought it would be better to put the deprecated install plans after the data source info but can move it above again if you want!